pool/src/lib.rs

```diff
@@ -79,6 +82,36 @@ impl LendingPoolTrait for LendingPool {
         read_reserve(&env, asset).ok()
     }
 
+    /// Sets the price feed oracle address.
+    ///
+    /// # Arguments
+    ///
+    /// - feed - The address of the price feed oracle.
+    ///
+    /// # Panics
+    ///
+    /// - Panics with `Uninitialized` if the admin key is not exist in storage.
+    /// - Panics if the caller is not the admin.
+    ///
+    fn set_price_feed(env: Env, feed: Address) -> Result<(), Error> {
+        Self::ensure_admin(&env)?;
+        // ensure_contract(env, feed)?;
```

Comment:
We can try to create PriceFeeder here, it'll panic on creation, it Address is not a contract, that implement `PriceFeedTrait`

Reply:
Replaced with the PriceProvider initialization.